playwright/pages/bankAccount.page.ts

```diff
@@ -0,0 +1,32 @@
+import { Page } from "@playwright/test";
+import {expect, test} from "@playwright/test";
+import { SideBlock } from "./sideBlock";
+import { AccountPage } from "./account.page";
+
+export class BankAccountPage {
+    readonly page : Page;
+    readonly headerLocator = 'transaction-detail-header';
+
+    constructor(page : Page) {
+        this.page = page;
+    }
+
+    async deleteBankAccount(id: string, name: string) {
+        await this.page.getByTestId(`bankaccount-list-item-${id}`).getByTestId('bankaccount-delete').click();
+        await expect(this.page.getByTestId(`bankaccount-list-item-${id}`)).toHaveText(`${name} (Deleted)`)
+    }
+
+    async createAccount(bank: string, routingNum: string, accountNum: string) {
+        await this.page.getByTestId('bankaccount-new').click();
+        await this.page.locator('#bankaccount-bankName-input').pressSequentially(bank);
+        await this.page.locator('#bankaccount-routingNumber-input').pressSequentially(routingNum);
+        await this.page.locator('#bankaccount-accountNumber-input').pressSequentially(accountNum);
```

Comment:
same comment about selectors, the best way to create WebElement separate

Reply:
I'm just not sure do we really need to move locator to class level if it's used in one function only. But let's discuss this.